odxtools/database.py

```diff
@@ -9,9 +9,8 @@
 from .diaglayer import DiagLayer
 from .diaglayercontainer import DiagLayerContainer
 from .globals import logger
-from .nameditemlist import NamedItemList
+from .nameditemlist import NamedItemList, short_name_as_key
 from .odxlink import OdxLinkDatabase
-from .utils import short_name_as_id
 
 
 def version(v: str):
```

Comment:
this is a drive-by (sorry), but this function should probably become a static member of the `Database` class and be renamed to something like `parse_model_version()`?

Reply:
Out of PR scope, current PR overloaded as is